st2common/tests/unit/test_crypto_utils.py

```diff
@@ -14,33 +14,227 @@
 # limitations under the License.
 
 from __future__ import absolute_import
-from keyczar.keys import AesKey
-from unittest2 import TestCase
 
+import os
+
+import six
+import json
+import binascii
 
-import st2common.util.crypto as crypto_utils
+import unittest2
+from unittest2 import TestCase
 from six.moves import range
+from cryptography.exceptions import InvalidSignature
+
+from st2common.util.crypto import KEYCZAR_HEADER_SIZE
+from st2common.util.crypto import AESKey
+from st2common.util.crypto import read_crypto_key
+from st2common.util.crypto import symmetric_encrypt
+from st2common.util.crypto import symmetric_decrypt
+from st2common.util.crypto import keyczar_symmetric_decrypt
+from st2common.util.crypto import keyczar_symmetric_encrypt
```

Comment:
Can be rewritten to:

```python
import st2common.util.crypto as crypto
keyczar_symmetric_encrypt = crypto.keyczar_symmetric_encrypt
```

so you don't have to export `keyczar_symmetric_encrypt`.

Same story with `keyczar_symmetric_decrypt`. See my companion comment in `st2common/utils/crypto.py`.

Reply:
See my comment above - personal preference and I think it's easier to glance at the top of the module to see what functions / classes a particular module imports and uses.